### ISSUE TITLE
live-dashboard: Only send verification emails to new users

### ISSUE THREAD

**Issue by dannypernik** [opencurrents/opencurrents#75]
Max received a verification email after we added his email to live-dashboard despite already being registered.

**Comment by nickolashe** [comment]
ah yes, good catch!

**Comment by nickolashe** [comment]
Actually @dannypernik do you where this took place? In production? I've just merged and deployed the latest copy of develop. This should not happen any more.

**Comment by MaxTattenbach** [comment]
@nickolashe  Danny,  sent added me to an event with my email to which I had already registered with.  I then received the validation email to my gmail, and invalidated my previous password. 
Max

**Comment by nickolashe** [comment]
@MaxTattenbach @dannypernik I don't think I am following still. Do we think there is an issue or not? If we do, can you please state whether you saw on the production site or locally and the sequence of steps you took so I can reproduce.

**Comment by nickolashe** [comment]
bump. do we still think this is an issue?

**Comment by dannypernik** [comment]
@nickolashe It was confusing for us too. It seemed that Max was in fact getting a confirmation email when it was added to live-dashboard locally, and it may have locked him out. However, I just uncommented the "Add volunteer" inputs locally to test, and I have not received a confirmation email. We may want to watch for this if we do decide to use the "add volunteer" feature later, but given that I'm not seeing it now and we're not using this feature anyway, I'm closing it.
